### PR TITLE
Add new output variables for ZoneHVAC:IdealLoadsAirSystem supply and outdoor air flow rates

### PR DESCRIPTION
**Pull request by JasonGlazer**
to show the outdoor and system air flow rates as both mass and volumetric units. Fixing #4325.

### CODE REVIEW

**Comment by JasonGlazer** [comment]
Looks like this commit is mostly about documentation changes but those are just because my editor got rid of a bunch of trailing spaces. The code change are in the .hh and .cc files.

**Comment by JasonGlazer** [comment]
@rraustad or @mjwitte can one of you look through these changes and make sure I put the code in the right place in the .cc file. I have not messed with the purchair related code before and want to make sure I am not missing something.

File: src/EnergyPlus/PurchasedAirManager.cc [hunk 1]
```diff
@@ -700,6 +705,12 @@ namespace PurchasedAirManager {
 
 			SetupOutputVariable( "Zone Ideal Loads Hybrid Ventilation Available Status []", PurchAir( PurchAirNum ).AvailStatus, "System", "Average", PurchAir( PurchAirNum ).Name );
 
+			//air flows
```
Comment (mjwitte) [inline src/EnergyPlus/PurchasedAirManager.cc:708]: @JasonGlazer Looks like the indentation is wrong around here.  Maybe spaces instead of tabs?

**Comment by mjwitte** [comment]
@JasonGlazer Otherwise, the changes look OK.  You should be able to verify by comparing to the equivalent System Node . . . outputs.

File: src/EnergyPlus/PurchasedAirManager.cc (hunk 1 above)
Comment (JasonGlazer) [inline src/EnergyPlus/PurchasedAirManager.cc:708]: @mjwitte it looks ok to me. The bracket above the line you commented on is from a much earlier set of lines.
The file uses tabs but all the files use tabs on my machine (is that correct?).
Do you really want me to use spaces instead?

Comment (mjwitte) [inline src/EnergyPlus/PurchasedAirManager.cc:708]: No - tabs are correct.  Maybe github lied to me when I expanded this.

Comment (mjwitte) [inline src/EnergyPlus/PurchasedAirManager.cc:708]: When you look at this on github, line 706 is left of 704.  Doesn't seem correct.

Comment (JasonGlazer) [inline src/EnergyPlus/PurchasedAirManager.cc:708]: Weird, it doesn't render that way for me on Github

Comment (mjwitte) [inline src/EnergyPlus/PurchasedAirManager.cc:708]: And it doesn't render that way for me when I look in Chrome instead of IE.  Fascinating.  Never mind.

**Comment by JasonGlazer** [comment]
Reviewed the many diffs and they all seem reasonable. The audit and rdd diffs are directly due to the addition of four new output variables. The differences in the tabular HTML file is just the build number on the input verification report. It should probably not be shown as a difference - @Myoldmopar would this be possible?

**Comment by JasonGlazer** [comment]
I also did some checks of the new supply air mass flow rate and volume flow rate compared to system node values and they match.

**Comment by JasonGlazer** [comment]
@Myoldmopar and @mjwitte I believe this is ready to be merged.

File: doc/src/docs/InputOutputReference/01c-InputOutputReference.md [hunk 2]
```diff
@@ -7152,6 +7152,15 @@ An example of this object in an IDF is:
 
 * HVAC,Average,Zone Ideal Loads Hybrid Ventilation Available Status []
 
+* HVAC,Average,Zone Ideal Loads Outdoor Air Mass Flow Rate [kg/s]
+
```
Comment (mjwitte) [inline doc/src/docs/InputOutputReference/01c-InputOutputReference.md:7156]: @JasonGlazer Please add the variable descriptions further down.  You can adapt the text from the equivalent System Node flow variable descriptions.  Be sure to pull this branch before editing - I've merged in develop and resolved a conflict today.

File: tst/EnergyPlus/unit/PurchasedAirManager.unit.cc [hunk 3]
```diff
@@ -107,3 +110,55 @@ TEST( SizePurchasedAirTest, Test1 )
...
+	EXPECT_EQ( PurchAir( 1 ).MaxHeatSuppAirTemp, 50.0);
+	EXPECT_EQ( PurchAir( 1 ).MinCoolSuppAirTemp, 13.0 );
+	EXPECT_EQ( PurchAir( 1 ).MaxHeatSuppAirHumRat, 0.015 );
+	EXPECT_EQ( PurchAir( 1 ).MinCoolSuppAirHumRat, 0.009 );
+	EXPECT_EQ( PurchAir( 1 ).HeatingLimit, NoLimit );
+	EXPECT_EQ( PurchAir( 1 ).CoolingLimit, NoLimit );
+	EXPECT_EQ( PurchAir( 1 ).DehumidCtrlType, ConstantSupplyHumidityRatio );
+	EXPECT_EQ( PurchAir( 1 ).HumidCtrlType, ConstantSupplyHumidityRatio );
+
+}
```
Comment (mjwitte) [inline tst/EnergyPlus/unit/PurchasedAirManager.unit.cc]: Also, this needs a newline after the last line.

**Comment by mjwitte** [comment]
Reviewed and tested, ready to merge once CI tests are complete.  @Myoldmopar Any comments?

**Comment by Myoldmopar** [comment]
No significant comment.  The code looks perfect.  And it looks like your editor cleaned up lots of whitespace changes in the docs.  (Easy enough to filter using Github by appending ?w=1 to the end of the URL when viewing the files changed tab.).  The only comment I'd say is in the future, instead of adding more variables to the [unused member constructor](https://github.com/NREL/EnergyPlus/pull/5293/files?w=1#diff-b5650bce0642a99343efa9d4365d99b4L308), just take the member constructor out.  

Merge away when you are happy with CI, or I can merge if you are looking for a volunteer.

**Pull request merged**